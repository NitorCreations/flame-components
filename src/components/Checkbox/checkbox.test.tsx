import * as React from 'react';
import { shallow } from 'enzyme';
import CheckBox from './CheckBox';

describe('TextInput', () => {
  it('Should render correctly', () => {
    const component = shallow(<CheckBox />);

    expect(component).toMatchSnapshot();
  });

  it('Should be correctly enabled', () => {
    const component = shallow(<CheckBox disabled={false} />);

    expect(component).toMatchSnapshot();
  });

  it('Should be correctly disabled', () => {
    const component = shallow(<CheckBox disabled />);

    expect(component).toMatchSnapshot();
  });
});