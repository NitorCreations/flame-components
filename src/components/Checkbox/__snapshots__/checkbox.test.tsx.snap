// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput Should be correctly disabled 1`] = `
<label
  className="flame-checkbox  disabled"
>
  <input
    checked={false}
    disabled={true}
    type="checkbox"
  />
  <span />
</label>
`;

exports[`TextInput Should be correctly enabled 1`] = `
<label
  className="flame-checkbox  "
>
  <input
    checked={false}
    disabled={false}
    type="checkbox"
  />
  <span />
</label>
`;

exports[`TextInput Should render correctly 1`] = `
<label
  className="flame-checkbox  "
>
  <input
    checked={false}
    type="checkbox"
  />
  <span />
</label>
`;
